core/common/src/test/GenericInstanceFilter.test.ts

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { GenericInstanceFilter, GenericInstanceFilterRule, GenericInstanceFilterRuleValue } from "../GenericInstanceFilter";
+
+describe("GenericInstanceFilterRuleValue", () => {
+  it("'isPoint2d' returns correct result", () => {
```

Comment:
We're not testing `isPoint2d` against a `Point3d` value, so it's not entirely clear what to expect from it. Looking at the implementation, I see it returns `true` for `Point3d` values - is that intended? If yes, please mention that in `isPoint2d` comment. If no, then the function needs to be fixed. An additional check for `Point3d` value in this test is needed either way, IMO.

Reply:
It wasn't intentional I just wanted to keep implementation simple. I don't see a problem with returning `true` for `Point3d` so updated test and added comment on `isPoint2d`.